usr/src/lib/libict_pymod/ict.py

```diff
@@ -2347,6 +2350,36 @@ def create_new_user(self, gcos, login, pw, gid, uid):
 
         return return_status
 
+    def copy_generated_files(self, login, uid, gid):
+        '''ICT - copy generated files from /jack to user home
+        '''
+        _register_task(inspect.currentframe())
+        
+        return_status = 0
+        _dbg_msg('copying generated user files to home directory under: ' + self.basedir)
+        
+        if not login:
+            _dbg_msg('No login specified')
+            return return_status
+        
+        dest_dir = self.basedir + '/export/home/' + login
```

Comment:
Nitpick, [os.path.join](https://docs.python.org/2/library/os.path.html#os.path.join) would be nicer here.